qa/rpc-tests/mn_tickets.py

```diff
@@ -512,6 +518,76 @@ def personal_royalty_initialize_tests(self):
+        artist_last_names=('Johnson','Smith','Williams', 'Ecclestone', 'Schumacher', 'Faye', 'Counts', 'Wesley')
+        letters = string.ascii_letters
+
+        # initialize hash base strings or lists
+        thumbnail_to_be_hashed = ''.join(random.choice(letters) for i in range(10))
+        data_to_be_hashed = ''.join(random.choice(letters) for i in range(10))
+        fingerprints_to_be_hashed = ''.join(random.choice(letters) for i in range(10))
+        rq_oti = ''.join(random.choice(letters) for i in range(12))
+        rq_ids_to_be_hashed = ""
+        for _ in range (5):
+            rq_ids_to_be_hashed += (''.join(random.choice(letters) for i in range(10)))
+        
+        # encode the string
+        encoded_thumbnail = thumbnail_to_be_hashed.encode()
+        encoded_data = data_to_be_hashed.encode()
+        encoded_fingerprint = fingerprints_to_be_hashed.encode()
+        encoded_rq_ids = rq_ids_to_be_hashed.encode()
+        
+        # create sha3-256 hash objects
+        obj_sha3_256_thumbnail = hashlib.sha3_256(encoded_thumbnail)
+        obj_sha3_256_data = hashlib.sha3_256(encoded_data)
+        obj_sha3_256_fingerprint = hashlib.sha3_256(encoded_fingerprint)
+        obj_sha3_256_rq_ids = hashlib.sha3_256(encoded_rq_ids)
+
+        art_ticket_json = {
+            "artist_name": "".join(random.choice(artist_first_names)+" "+random.choice(artist_last_names)),
+            "artwork_title": ''.join(random.choice(letters) for i in range(10)),
+            "artwork_series_name": ''.join(random.choice(letters) for i in range(10)),
+            "artwork_keyword_set": ''.join(random.choice(letters) for i in range(10)),
+            "artist_website": ''.join(random.choice(letters) for i in range(10)),
+            "artist_written_statement": ''.join(random.choice(letters) for i in range(10)),
+            "artwork_creation_video_youtube_url": ''.join(random.choice(letters) for i in range(10)),
+            "thumbnail_hash": obj_sha3_256_thumbnail.hexdigest(),    #hash of the thumbnail !!!!SHA3-256!!!!
+            "data_hash": obj_sha3_256_data.hexdigest(),         #hash of the image (or any other asset) that this ticket represents !!!!SHA3-256!!!!
+            "fingerprints_hash": obj_sha3_256_fingerprint.hexdigest(),       #hash of the fingerprint !!!!SHA3-256!!!!
+            "fingerprints": fingerprints_to_be_hashed,            #compressed fingerprint
+            "fingerprints_signature": ''.join(random.choice(letters) for i in range(20)), #signature on raw image fingerprint
+            "rq_ids": obj_sha3_256_rq_ids.hexdigest(), #[list of strings],//raptorq symbol identifiers -  !!!!SHA3-256 of symbol block!!!!
+            "rq_oti": rq_oti,    #raptorq CommonOTI and SchemeSpecificOTI
+            "rareness_score": str(random.randint(0, 1000)),   # 0 to 1000
+            "nsfw_score": str(random.randint(0, 1000)),   # 0 to 1000
+            "seen_score": str(random.randint(0, 1000)),   # 0 to 1000
+        }
+
```

Comment:
Do we need to test art_ticket after ticket registration?

Reply:
? as said I got exact task from @ayk to proceed like this. Dont know what is the point here.

Reply:
Could you please specify what you meant ? :)

Reply:
You created here app_ticket and that is the part of self.ticket used in "register" "art". I'm just asking if need to write tests with reading data back from app_ticket after registration. Or probably it is not related to this task.

Reply:
Yeah, i dont think it is the scope of this change, but since mock-network is updated based on this logic, mock-network repo can be checked!